### DISCUSSION TITLE
add nilesoft shell to directory opus context menu

### DISCUSSION THREAD

**Discussion by MahdiSh81** [moudey/Shell#544] [Q&A]
how can i make directory opus use nilesoft context menu instead of its default context menu

**Comment by RubicBG** [comment]
https://github.com/moudey/Shell/issues/39#issuecomment-1343691190
https://github.com/moudey/Shell/issues/324#issuecomment-1715673962

**Comment by JoyHak** [comment]
NS needs to be changed so that it can work in other applications...
Anyways I like the idea of integrating the beautiful Nilesoft shell into other file management applications: Xyplorer file manager, everything search, Flow launcher search, ...
